libraries/standard/mqtt/src/mqtt.c

```diff
@@ -1377,7 +1501,7 @@ MQTTStatus_t MQTT_ProcessLoop( MQTTContext_t * const pContext,
     uint32_t entryTimeMs = 0U, remainingTimeMs = timeoutMs, elapsedTimeMs = 0U;
     MQTTPacketInfo_t incomingPacket;
 
-    if( pContext != NULL )
+    if( ( pContext != NULL ) && ( pContext->callbacks.getTime != NULL ) )
```

Comment:
Update the log message below to reflect getTime non-NULL check above.

Reply:
Updated